single_cell_analysis/scripts/00-data.download.processing/data.download.processing.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# Pipeline to generate/download simulated and real single cell datasets and do data processing
+#
+# Usage: bash scripts/00-data.download.processing/data.download.processing.sh
+#
+# Output: log(or TPM) - normalized count files and files contain celltype information. 
+#         files are located at data/real_data for real datasets and data/simulate_data for simulated datasets
+
+# 1. Generate simulated datasets
+  Rscript splatter.simulation.R
```

Comment:
If this is supposed to be run in `scripts/00-data.download.processing/` I do not think these lines will work. Should the user first navigate into this folder and then execute the bash script? Alternatively can add the full path to the R script (and all other scripts below)

Reply:
Thanks ! added full path to the R script